libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -18,6 +18,7 @@
 //	Origin code by Michael Smith, Jordi Munoz and Jose Julio, DIYDrones.com
 //  Substantially rewritten for new GPS driver structure by Andrew Tridgell
 //
+#include <bitset>
```

Comment:
You can't use this on all the platforms we use and support. (Notably does not work on ChibiOS/ARM GCC 6.3/ARM GCC 4.9) You will need to directly do masking/bit manipulation.